monai/utils/stopperutils.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def stopping_fn_from_metric(metric_name):
+    """Returns a stopping function for ignite.handlers.EarlyStopping using the given metric name."""
+    def stopping_fn(engine):
+        return engine.state.metrics[metric_name]
+    return stopping_fn
+
+def stopping_fn_from_loss():
+    """Returns a stopping function for ignite.handlers.EarlyStopping using the loss value."""
+    def stopping_fn(engine):
+        return -engine.state.loss.item()
```

Comment:
Where did you define "state.loss"? I don't think it's a pre-defined item in engine.state.
Thanks.

Reply:
@Nic-Ma this should be engine.state.output. Fixed.